### PR TITLE
updates to API readiness checklist

### LINKED ISSUE #110: Update comments column in API-readiness-checklist.md
**Problem description**
The comments column is too open  and can be used better. 

**Expected action**

- Rename the "Comments" column  to "Reference information", 
- add the Commonalities and ICM version 
- update the explanation on the wiki

**Additional context**

### LINKED ISSUE #113: Fix link to point to new wiki in the API-Readiness-Checklist.md
**Problem description**
The move to the cloud wiki has broken the link in the API-Readiness-Checklist.md template.

**Expected behavior**
Change link to point to the new cloud wiki

### PR DESCRIPTION
**Pull request by tanjadegroot**
#### What type of PR is this?

Add one of the following kinds:
* documentation

#### What this PR does / why we need it:
Updates to the API readiness checklist following experience and feedback from the Fall24 meta-release

#### Which issue(s) this PR fixes:

Fixes #110 
Fixes #113 

```
docs

```

### CODE REVIEW

File: documentation/API-Readiness-Checklist.md [hunk 1]
```diff
@@ -2,26 +2,27 @@
...
+| Nr | API release assets  | alpha | release-candidate |  initial<br>public | stable<br> public | Status | Reference information |
 |----|----------------------------------------------|:-----:|:-----------------:|:-------:|:------:|:----:|:----:|
 |  1 | API definition                               |   M   |         M         |    M    |    M   |      | link |
-|  2 | Design guidelines from Commonalities applied |   O   |         M         |    M    |    M   |      |      |
-|  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |      |      |
-|  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |      |      |
+|  2 | Design guidelines from Commonalities applied |   O   |         M         |    M    |    M   |      | Comm. release nr |
+|  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |      | ICM release nr |
+|  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |      | x.y.z |
```
Comment (hdamker) [inline documentation/API-Readiness-Checklist.md:10]: x.y.z is doubling the information from line 3 here ("api-version"). And there could be other formats as `x.y.z`, e.g. `x.y.z-rc.n`
I wouldn't require here a reference information.

File: documentation/API-Readiness-Checklist.md [hunk 2]
```diff
@@ -2,26 +2,27 @@
...
 |  5 | API documentation                            |   M   |         M         |    M    |    M   |      | link |
 |  6 | User stories                                 |   O   |         O         |    O    |    M   |      | link |
 |  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |      | link |
 |  8 | Enhanced API test cases & documentation      |   O   |         O         |    O    |    M   |      | link |
 |  9 | Test result statement                        |   O   |         O         |    O    |    M   |      | link |
-| 10 | API release numbering convention applied     |   M   |         M         |    M    |    M   |      |      |
+| 10 | API release numbering convention applied     |   M   |         M         |    M    |    M   |      | rx.y |
```
Comment (hdamker) [inline documentation/API-Readiness-Checklist.md:16]: Similar as previous comment.

Alternative would be to reduce line 3 to api-name.

File: documentation/API-Readiness-Checklist.md [hunk 3]
```diff
@@ -2,26 +2,27 @@
...
-| 12 | Previous public release was certified        |   O   |         O         |    O    |    M   |      |      |
+| 12 | Previous public release was certified        |   O   |         O         |    O    |    M   |      | link |
 
 To fill the checklist:
 - in the line above the table, replace the api-name, api-version and the rx.y by their actual values for the current API version and release.
 - in the Status column, put "Y" (yes) if the release asset is available or fulfilled in the current release, a "N" (no) or a "tbd". Example use of "tbd" is in case an alpha or release-candidate API version does not yet provide all mandatory assets for the release.
-- in the Comments column, provide the link to the asset once available, and any other relevant comments.
+- in the Reference information column, provide the links to the release asset once available, the applicable release numbers (not versions) of Commonalities and ICM, and any other relevant links or information.
+- For the point 12: you can copy the following link if your previous API version has been certified: [GSMA OGW Portal](https://www.open-gateway.com/operators-map).
```
Comment (hdamker) [inline documentation/API-Readiness-Checklist.md:24]:
```suggestion
- For the point 12: you can copy the following link if your previous API version has been certified by at least one operator: [GSMA OGW Portal](https://www.open-gateway.com/operators-map).
```

**Review by hdamker — COMMENTED** [review]
See comments, I would avoid to double the information in the file

File: documentation/API-Readiness-Checklist.md (hunk 1 above)
Comment (tanjadegroot) [reply documentation/API-Readiness-Checklist.md:10]: indeed, will remove it

**Comment by tanjadegroot** [comment]
Points taken. we also have an issue with the API specification - people tend to put a local link (which exapnds to a url with "main". should we make that field empty too ?

**Comment by hdamker** [comment]
> Points taken. we also have an issue with the API specification - people tend to put a local link (which expands to a url with "main". should we make that field empty too ?

No, that are [relative links](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#relative-links), they expand to main only if you are on the main branch: "GitHub will automatically transform your relative link or image path based on whatever branch you're currently on, so that the link or path always works" ... which works also for release tag trees and even for forked repositories - which is exactly what we need (and also have the benefit that they only need to be changed if file names or directory structure changes).

They will not work outside of GitHub, e.g. in a local clone. But there they are still describing the correct path to the right file from the root of the repository.

**Review by hdamker — APPROVED** [review]

**Review by rartych — APPROVED** [review]
LGTM

**Review by hdamker — APPROVED** [review]

**Comment by rartych** [comment]
During last TSC Mark Cornall introduced a new version of API deployment map:  https://open-gateway.gsma.com/
 that is planned to evolve into developer portal.
Maybe this address should be referenced in the note to p.12? 
But we can change it later - currently both versions are online: https://www.open-gateway.com/operators-map  and https://open-gateway.gsma.com/

**Comment by hdamker** [comment]
> During last TSC Mark Cornall introduced a new version of API deployment map: https://open-gateway.gsma.com/ that is planned to evolve into developer portal. Maybe this address should be referenced in the note to p.12? But we can change it later - currently both versions are online: https://www.open-gateway.com/operators-map and https://open-gateway.gsma.com/

@rartych Good point. The official [OGW page](https://www.gsma.com/solutions-and-impact/gsma-open-gateway/) is already linking to the [new map version](https://open-gateway.gsma.com/), therefore I suggest to change also on our side now.

**Comment by tanjadegroot** [comment]
OK, I was not sure that the new map is replacing the other but Herbert's comment seems to inticate this is the case. 
I will update to the new link.

**Comment by tanjadegroot** [comment]
> > Points taken. we also have an issue with the API specification - people tend to put a local link (which expands to a url with "main". should we make that field empty too ?
> 
> No, that are [relative links](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#relative-links), they expand to main only if you are on the main branch: "GitHub will automatically transform your relative link or image path based on whatever branch you're currently on, so that the link or path always works" ... which works also for release tag trees and even for forked repositories - which is exactly what we need (and also have the benefit that they only need to be changed if file names or directory structure changes).
> 
> They will not work outside of GitHub, e.g. in a local clone. But there they are still describing the correct path to the right file from the root of the repository.

Update the chacklist to refer to "relative links"

**Review by soadeyemo — APPROVED** [review]

**Pull request merged**